cognite_toolkit/_cdf_tk/resource_classes/capabilities.py

```diff
@@ -510,6 +510,18 @@ class LegacyGenericsAcl(Capability):
     scope: AllScope
 
 
+class StreamsAcl(Capability):
+    _capability_name = "streamsAcl"
+    actions: list[Literal["READ", "CREATE", "DELETE"]]
+    scope: AllScope
+
+
+class StreamRecordsAcl(Capability):
+    _capability_name = "streamRecordsAcl"
+    actions: list[Literal["READ", "WRITE"]]
+    scope: AllScope | SpaceIDScope
```

Comment:
Do we support SpaceIDScope? I cannot see that in the proto files for the servicehttps://github.com/cognitedata/infrastructure/blob/436780675594a56c20110e0df5905c4b0bb147fa/lib/proto/api_acls/cogmono/api_acls/streams_acl/streams_acl.proto